TicTacToe2/TicTacToeGame.cs

```diff
@@ -24,115 +28,251 @@ public void Play()
+                    Console.WriteLine("\nPlease enter a number.");
+                }
+                else if (gridNumber <= 0 || gridNumber > 2)
+                {
+                    correctGrid = false;
+                    Console.WriteLine("\nPlease choose 1 or 2.");
+                }
+                else
+                {
+                    correctGrid = true;
+                }
+            }
+        }
+
         private void DrawBoard()
         {
-            Console.Clear();
-            Console.WriteLine($"\n\t {space[0]} | {space[1]} | {space[2]}");
-            Console.WriteLine("\t-----------");
-            Console.WriteLine($"\t {space[3]} | {space[4]} | {space[5]}");
-            Console.WriteLine("\t-----------");
-            Console.WriteLine($"\t {space[6]} | {space[7]} | {space[8]}");
+            if (gridNumber == 1)
+            {
+                Console.Clear();
+                Console.WriteLine($"\n\t {space[0]} | {space[1]} | {space[2]}");
+                Console.WriteLine("\t-----------");
+                Console.WriteLine($"\t {space[3]} | {space[4]} | {space[5]}");
+                Console.WriteLine("\t-----------");
+                Console.WriteLine($"\t {space[6]} | {space[7]} | {space[8]}");
+            }
+            else
+            {
+                Console.Clear();
+                Console.WriteLine($"\n\t  {space[0]} | {space[1]} | {space[2]} | {space[3]}");
+                Console.WriteLine("\t--------------------");
+                Console.WriteLine($"\t  {space[4]} | {space[5]} | {space[6]} | {space[7]}");
+                Console.WriteLine("\t--------------------");
+                Console.WriteLine($"\t  {space[8]} | {space[9]} | {space[10]} | {space[11]}");
+                Console.WriteLine("\t--------------------");
+                Console.WriteLine($"\t  {space[12]} | {space[13]} | {space[14]} | {space[15]}");
+            }
+
         }
 
         private void GetUserInput()
         {
             string choiceText;
             bool validInput = false;
-            bool legalMove = false;
 
-            while (!validInput && !legalMove)
+            while (!validInput)
             {
                 Console.WriteLine($"\nIt is player {player}'s turn.");
                 Console.Write("Pick a number from the board to place your mark: ");
                 choiceText = Console.ReadLine();
 
-                bool isParsable = Int32.TryParse(choiceText, out int choice);
-                //choice--;
-                if (!isParsable)
+                bool isParsable = Int32.TryParse(choiceText, out choice);
+                
+                if (!isParsable && !correctNumber)
                 {
-                    validInput = false;
                     Console.WriteLine("Please enter a number.");
+                    validInput = false;
                 }
-                else if (choice <= 0 || choice > 9)
+                else if (correctNumber = CorrectChoice())
```

Comment:
This line is causing the result of CorrectChoice() to be assigned to correctNumber. Be sure to use == when comparing values. This is a common mistake.